### DISCUSSION TITLE
Implementing SAS fcs discrim method with mice

### DISCUSSION THREAD

**Discussion by VictorPorcelli** [amices/mice#543] [Q&A]
Hi there. I've been working with mice to impute data at my workplace, which recently transitioned from SAS to R. Previously, SAS's PROC MI was used to impute data. One particular difference we've noticed is the imputation of binary variables for complex, large datasets. I know best practices are to simplify your imputation model, which is something we are working to do. However, we've still noticed that using 'fcs discrim' with classeffects to impute binary variables in SAS seems to produce better results than using the logreg method in R with mice.

I've attempted to look into the two methods and understand them better, but I do not have a degree in statistics and some of it goes over my head. Does anyone know of a way to implement fcs discrim or a similar method to impute binary variables using mice? Please let me know any thoughts or suggestions -- thanks in advance.

**Comment by stefvanbuuren** [comment]
`mice` has a method called `mice.impute.lda()` that implements linear discriminant analysis.
